test/framework/toy_build.py

```diff
@@ -1532,10 +1533,27 @@ def test_minimal_toolchains(self):
         # this test doesn't check for anything specific to using minimal toolchains, only side-effects
         self.test_toy_build(extra_args=['--minimal-toolchains'])
 
+    def test_reproducability(self):
+        """Test toy build produces expected reproducability files"""
+        # use the easyblock with inheritance to fully test
+        self.test_toy_build(extra_args=['--minimal-toolchains', '--easyblock=EB_toytoy'])
```

Comment:
I happened to notice that the `--easyblock` option is not recorded in the reproducible easyconfig. Not sure where I would do this...

Reply:
Fixed in this PR